.travis.yml

```diff
@@ -4,12 +4,24 @@ language: go
 go:
   - 1.9.2
 go_import_path: github.com/mysterium/node
+
+env:
+  global:
+    - CACHE_ROOT_DIR=$HOME/cache
+    - TRAVIS_TOOLS_PATH=$CACHE_ROOT_DIR/travis_tools
```

Comment:
This is confusing - `TRAVIS_TOOLS_PATH` refers like it's tools given by Travis.
`BUILD_TOOLS_PATH` ?

Reply:
fixed